src/js/easymde.js

```diff
@@ -2048,7 +2049,8 @@ EasyMDE.prototype.render = function (el) {
     if (options.status !== false) {
         this.gui.statusbar = this.createStatusbar();
     }
-    if (options.autosave != undefined && options.autosave.enabled === true) {
+    var autosaveEnabled = options.autosave != undefined && options.autosave.enabled === true;
+    if (autosaveEnabled) {
```

Comment:
What's the purpose of this change?

Reply:
@Ionaru well basically I reused change listener, so reused this check, though you ar right,  rolled back now